server.R

```diff
@@ -164,10 +164,10 @@ shinyServer(function(input, output, session) {
         )
         file_namedList <<- list2Vector(file_list)
 
-        manifest_url <- metadata_model$getModelManifest(paste0(
-          config$community,
-          " ", input$dropdown_template
-        ), template_schema_name, filenames = as.list(names(file_namedList)))
+        manifest_url <- metadata_model$getModelManifest(paste0(config$community," ", input$template_type), 
+                                                        template_type, 
+                                                        filenames = as.list(filename_list),
+                                                        datasetId = folder_synID)
```

Comment:
In the `shin-server-develop` branch, some variables' names are altered. 
- `filename_list` is removed (which was calling the `names(file_namedList)`.
- `input$template_type` is changed to `input$dropdown_template`
- `template_type` is changed to `template_schema_name`

Can you change this to:
```
metadata_model$getModelManifest(paste0(config$community, " ", input$dropdown_template),  
  template_schema_name, 
  filenames = as.list(names(file_namedList)),
  datasetId = folder_synID
)
```
Sorry about the inconvenience.

Reply:
No problem, I should have checked first! 
```suggestion
 metadata_model$getModelManifest(paste0(config$community, " ", input$dropdown_template),  
  template_schema_name, 
  filenames = as.list(names(file_namedList)),
  datasetId = folder_synID
)
```